bpa-services/src/main/resources/application.properties

```diff
@@ -34,6 +36,7 @@ spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.Str
 spring.kafka.consumer.group-id=egov-bpa-services
 spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
 spring.kafka.producer.value-serializer=org.springframework.kafka.support.serializer.JsonSerializer
+spring.kafka.listener.missing-topics-fatal=false
```

Comment:
Add `management.endpoints.web.base-path=/` also for health check backward compatibility

Reply:
management.endpoints.web.base-path=/
management.endpoints.web.path-mapping.health=healthcheck

adding above two